app/journals/[id]/page.tsx

```
@@ -73,37 +76,48 @@ export default function JournalDetailPage() {
     )
   }
 
+  // Build the submission URL with the journal path for deep-linking
+  const [activeTab, setActiveTab] = useState("about")
```

Biome (2.4.6)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

(lint/correctness/useHookAtTopLevel)